r2/r2/lib/system_messages.py

```diff
@@ -130,10 +145,70 @@ def send_ban_message(subreddit, mod, user, note=None, days=None, new=True):
 
     message += "\n\n" + ("you can contact the moderators regarding your ban "
         "by replying to this message. **warning**: using other accounts to "
-        "circumvent a subreddit ban is considered a violation of reddit's "
+        "circumvent a %(ban_type)s ban is considered a violation of reddit's "
         "[site rules](/rules) and can result in being banned from reddit "
-        "entirely.")
+        "entirely." % {"ban_type": ban_type})
 
     item, inbox_rel = Message._new(mod, user, subject, message, request.ip,
         sr=subreddit, from_sr=True)
     queries.new_message(item, inbox_rel, update_modmail=False)
+
+
+def send_mod_permission_message(subreddit, changer, mod, oldperms,
```

Comment:
Oh good! Then that resolves my worries; I suspected admins had some tools that weren't immediately apparent. :+1:  Thank you for your swift reply!